src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -195,9 +194,9 @@ public final void testPathsDirectoryWithNestedFile() throws IOException {
                 TestRootModuleChecker.isProcessed());
         final List<File> filesToCheck = TestRootModuleChecker.getFilesToCheck();
         assertThat("There more files to check then expected",
-                filesToCheck.size(), is(1));
+                filesToCheck.size(), is(9));
         assertThat("The path of file differs from expected",
-                filesToCheck.get(0).getAbsolutePath(), is(getPath(FLAWLESS_INPUT)));
+                filesToCheck.get(5).getAbsolutePath(), is(getPath(FLAWLESS_INPUT)));
```

Comment:
I like we increase number of files, so `file` won't be a problem again.
However I am very concerned about using direct index `5` to find this specific input.
Do we actually sort the files list result? nondex should fail if we don't.